app/views/devise/two_step_verification/show.html.erb

```diff
@@ -92,14 +92,14 @@
                     placeholder: 'Enter 6-digit code',
                     autocomplete: 'off' %>
                 </div>
-                <button name="submit" id="submit_code" class="btn btn-lg btn-success">
+                <button name="submit" id="submit_code" class="btn btn-lg btn-success js-track" data-track-label="finish">
```

Comment:
If we don't use a custom label we'll capture both scenarios. Is that preferable?